### PR TITLE
SLCReader ported to python.

### PR DESCRIPTION
**Pull request by pavankumarbn**

### CODE REVIEW

**Comment by ajpmaclean** [comment]
Three comments:
1) Can you please rewrite `SLCReader.py` using [CutWithScalars](https://lorensen.github.io/VTKExamples/site/Python/VisualizationAlgorithms/CutWithScalars/) an an example template. We have standardised on using  `def main()` and `def get_program_parameters()`.
2) When you have done this, you can drop in the code found in snippets to generate the required `Test*.png` file. See: [WriteImage](https://lorensen.github.io/VTKExamples/site/Python/Snippets/WriteImage/), once the image is generated, remove the snippet code used to generate the image.
3) If you need your name and attribution, put this in the SLCReader.md file, when merged, git already tracks contributors.

Also, since vtkNamedColors was introduced in VTK version 6+, change these lines:
```python
if vtk.VTK_MAJOR_VERSION <= 5:
    mapper.SetInput(reader.GetOutput())
else:
    mapper.SetInputConnection(reader.GetOutputPort())
```
to just:
```python
mapper.SetInputConnection(reader.GetOutputPort())
```

**Comment by pavankumarbn** [comment]
@ajpmaclean Thanks for the review. I will do the changes.

File: src/Python/IO/SLCReader.py [hunk 1]
```diff
@@ -1,53 +1,63 @@
...
+        epilogue = ''''''
+        parser = argparse.ArgumentParser(description=description, epilog=epilogue, formatter_class=argparse.RawDescriptionHelpFormatter)
+        parser.add_argument('filename1', help='vw_knee.slc.')
+        args = parser.parse_args()
+        return args.filename1
+
+if __name__ == '__main__':
+    main()
+    
+
```
Comment (ajpmaclean) [inline src/Python/IO/SLCReader.py]: Please remove the excess line at the end of the file.

**Comment by ajpmaclean** [comment]
We also need a new image as in point 2) of my original comment, please.

File: src/Python/IO/SLCReader.py (hunk 1 above)
Comment (pavankumarbn) [reply src/Python/IO/SLCReader.py]: @ajpmaclean I have removed the excess lines. Please review on the same.

**Comment by pavankumarbn** [comment]
> We also need a new image as in point 2) of my original comment, please.

Sorry. I will provide.

**Comment by ajpmaclean** [comment]
In `SLCReader.py` you removed too many blank lines and the end of a file should have just **one** blank line.

**Comment by pavankumarbn** [comment]
@ajpmaclean Sorry for the delayed response.  I have made and uploaded new snapshot, Please review on the same.

**Comment by ajpmaclean** [comment]
Thankyou for your contribution.

**Comment by pavankumarbn** [comment]
@ajpmaclean Thank you.

**Pull request merged**